### PR TITLE
Migrating references from dockerhub/ixiacom repo  to ghcr/otg

### PR DESCRIPTION
**Pull request by biplamal**

### CODE REVIEW

File: docs/deployments.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Ixia-c is distributed / deployed as a multi-container application consisting of
 * **traffic-engine** - Generates, captures and processes traffic from one or more network interfaces (on linux-based OS).
 * **app-usage-reporter** - (Optional) Collects anonymous usage report from controller and uploads it to Keysight Cloud, with minimal impact on host resources.
 
-All these services are available as docker image on [ixiacom repository](https://hub.docker.com/u/ixiacom). Please check [Ixia-c Releases](releases.md) to use specific versions of these images.
+All these services are available as docker image on [GitHub Open-Traffic-Generator repository](https://github.com/orgs/open-traffic-generator/packages). Please check [Ixia-c Releases](releases.md) to use specific versions of these images.
```
Comment (ankur-sheth) [inline docs/deployments.md:22]: ...as docker image on... --> as docker images on...

**Review by ankur-sheth — CHANGES_REQUESTED** [review]
Please also add an item on docs/news.md that starting now images will be posted on GHCR.  We will continue to post on Docker Hub for the next 3 months and then stop posting there completely.  
Once this PR is submitted, we should also announce the same on the Ixia-c slack so that people can migrate.

File: docs/deployments.md (hunk 1 above)
Comment (biplamal) [reply docs/deployments.md]: Done

**Review by ankur-sheth — COMMENTED** [review]
Still need to add a section to the news.md.

**Pull request merged**